### PR TITLE
Fusetools2 572 fixname space

### PR DESCRIPTION
**Pull request by apupier**

### CODE REVIEW

**Comment by apupier** [comment]
- adding a test starting an integration in a specific namespace
- fix refresh of Integration explorer view when updating namespace (detected while writing the test and required to write the test)

File: src/kamel.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ class KamelImpl implements Kamel {
 	}
 
 	async getPath(): Promise<string> {
-		return await baseKamelPath();
+		return (await baseKamelPath()).replace('\n', '');
```
Comment (bfitzpat) [inline src/kamel.ts:44]: Do we know where the \n was slipping into the path?

**Review by bfitzpat — APPROVED** [review]
Looks good - just wondering about the `\n`

File: src/kamel.ts (hunk 1 above)
Comment (apupier) [reply src/kamel.ts:44]: if I understood well, it is the "which" command that is used. the output is adding a \n to have the prompt going to a new line.

Comment (apupier) [reply src/kamel.ts:44]: wonderign if not better to replace in the baseKamelPath method then

Comment (bfitzpat) [reply src/kamel.ts:44]: Yeah, that's kind of what I was thinking too -- might be better to move up stream a bit further?

**Review by bfitzpat — APPROVED** [review]

File: src/kamel.ts (hunk 1 above)
Comment (apupier) [reply src/kamel.ts]: agree. done

**Pull request merged**